charter.md

```diff
@@ -53,15 +53,227 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Composition
+
+The Steering Committee is composed of seven (7) members.
+
+### Elections
+
+<!--
+
+- Consider explicitly mentioning SIG ContribEx's Elections subproject, once it
+exists.
+
+-->
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+In the instance that a general election is called as a result of the
+[dissolution](#dissolution) of the current committee, the following rules will
+apply:
+
+* members of the current committee will not be allowed to run in the new
+  election
+* a new committee will be bootstrapped with:
+  * the top four candidates serving a two-year term
+  * the next three candidates serving a one-year team
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected steering committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible. [Eligible voters][voter-eligibility] from the most
+recent election will vote in the special election i.e., eligibility will not be
+redetermined at the time of the special election. A committee member elected in
+a special election will serve out the remainder of the term for the person they
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
```

Comment:
I think it could be difficult to mandate that members of a dissolved committee commit to serve in this capacity. Is this para more about the new committee _trying_ to get continuity as opposed to trying to mandate participation from outgoing members?

Reply:
Yes, maybe we should add, at the end, "if a selected former member isn't willing or able to perform this duty then the incoming board may choose a replacement. If there are no willing members then the new board will just have to all watch the last N meeting recordings."